DeAceroDLabs/marketplace-frontend
# [DMP-691] Get price of a product

## What is done

[[DMP-691] Obtener precio de producto utilizando su article id](https://deacero.atlassian.net/browse/DMP-691)

## How is done

1. Added a new param to the Article object in the API file (articleId).
2. Modified the active sku state in the ProductDetailView to include the `articleId`.
3. Send the new `articleId` param as a request param to the service price.

## How to test

1. Run the project via `npm start`
2. Click on a product to be redirected to the ProductDetailView. I recommend clicking on the first product (Clavo 2” - 5”).
3. Select the properties to get an exisiting sku. If you selected the one I recommended, then the properties would be 
  - Tipo: Reforzado.
  - Largo: 51.
  - Calibre: 11.
  - Presentación: 25kg
4. Once these are selected, you will get the sku, and a request will be sent to the price service.
5. The request MUST NOT send the sku as a param, but the articleId which is different.
6. The price for a single item should be displayed.
